### PR TITLE
Missing links

### PR DESCRIPTION
**Pull request by skrthomas**
There are a couple of files where you're missing a link for "here". I think I've found the appropriate links, but you can evaluate my PR to be sure. Otherwise, in the permissions file, there were a few links that were redirecting from the wiki to the docs page. To avoid the experience of the redirect, I replaced with the link directly to the doc.

### CODE REVIEW

File: docs/docs/commands.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Commands
 slug: /commands
 ---
 
-The main plugin command you'll be using is `/nick` to change your nickname, but all of the commands in the plugin are explained below. This page will reference color codes a lot, you can read about color formatting here.
+The main plugin command you'll be using is `/nick` to change your nickname, but all of the commands in the plugin are explained below. This page will reference color codes a lot, you can read about color formatting [here](https://docs.adventure.kyori.net/minimessage/format#color).
```
Comment (Majekdor) [inline docs/docs/commands.md:7]: This should point to https://hexnicks.majek.dev/color-formatting instead of adventure docs.

**Review by Majekdor — CHANGES_REQUESTED** [review]
I appreciate the PR! I obviously meant to add these things but just forgot

File: docs/docs/commands.md (hunk 1 above)
Comment (skrthomas) [reply docs/docs/commands.md]: @Majekdor of course :) I was just driving by to see if there were any obvious fixes, like broken or missing links. The doc looks great! Nice work all around 🥳

**Review by Majekdor — APPROVED** [review]

File: docs/docs/commands.md (hunk 1 above)
Comment (Majekdor) [reply docs/docs/commands.md]: Thank you so much!!

**Pull request merged**